msteams-platform/concepts/design/understand-use-cases.md

```diff
@@ -5,47 +5,84 @@ description: Understand your use cases
 ms.topic: conceptual
 ms.author: anclear
 ---
+
 # Understand your use cases
 
-The Microsoft Teams platform offers a large variety of [extensibility points and UI elements](~/concepts/extensibility-points.md) your app can take advantage of. If you don't already have a good understanding of what is possible on the Teams platform, you should read that article first.
+The Microsoft Teams platform offers a large variety of [extensibility points and UI elements](../../concepts/extensibility-points.md) your app can take advantage of.
+> [!NOTE]
+> Before you start building your use cases, you must have a good understanding of Teams capabilities and what is possible on the Teams platform using them.
+
+Each method of interacting with your users has its strengths and weaknesses. Building an awesome Teams app is all about finding the right combination to meet your user's needs. If you are going to meet those needs, you first need to understand them.
+
+## Understand the problem
+
+Every good app has a core problem or a need it is trying to solve. Before you start building an app, you need to articulate what that problem is. At its heart, Teams is a collaboration platform, so apps that solve collaboration problems are a great fit. It is also a social platform, is natively cross-platform, sits at the heart of Office 365, and offers a personal canvas for you to create apps. In this social platform, there is a wide variety of needs that can be solved with a Teams app. You can solve wide variety of problems, provided you understand which one you are trying to solve. Before you start building an app, ask relevant questions, such as:
```

Comment:
We can use - "so apps that encourage effective collaboration are a great fit..." OR
"...so apps that bridge gaps in achieving effective collaboration are a great fit..."